src/AutoDelete.sh

```
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+set -exo pipefail
+
+main() {
+
+    export PATH=$PATH:/home/dnanexus/.local/bin  
+
+    sudo -H python3 -m pip install --no-index --no-deps packages/*
+    
+    dx download "$config_file" -o config_file
```

Shellcheck (0.10.0)

[warning] config_file is referenced but not assigned.

(SC2154)